newswires/client/src/Item.tsx

```diff
@@ -35,13 +39,21 @@ export const Item = ({
 						<EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
 							<EuiButtonIcon
 								iconType="arrowLeft"
-								onClick={handleDeselectItem}
-								aria-contro
+								onClick={handlePreviousItem}
+								aria-label="Previous story"
 							/>
-							<EuiFlexGroup
-								justifyContent="flexEnd"
-								alignItems="center"
-							></EuiFlexGroup>
+							<EuiButtonIcon
+								iconType="arrowRight"
+								onClick={handleNextItem}
+								aria-label="Next story"
+							/>
+							<EuiFlexGroup justifyContent="flexEnd" alignItems="center">
+								<EuiButtonIcon
```

Comment:
I wonder if we want to automatically move focus into this panel when it's opened, possibly onto the close button?

I think we had some similar behaviour before but I seem to have broken it at some point... 🤔